dags/common/cleanup.py

```diff
@@ -18,7 +20,37 @@ def convert_html_subsripts_to_latex(input):
 def clean_collaboration(input):
     return clean_whitespace_characters(input.replace("for the", ""))
 
+
 def remove_specific_tags(value, tags=None, attributes=None):
     tags = tags or []
     attributes = attributes or []
     return clean(value, tags=tags, attributes=attributes, strip=True)
+
+
+class RemoveUnneccessaryTagsFilter(Filter):
```

Comment:
because the class is targetting specifically `label` WDYT to rename it to something like: `RemoveLabelTagsFilter`